src/common/components/repository-row/index.js

```diff
@@ -14,7 +14,13 @@ const RepositoryRow = (props) => {
     <Row>
       <Data col="30"><Link to={ `/${fullName}/builds` }>{ fullName }</Link></Data>
       <Data col="20"> </Data>
```

Comment:
@bartaz @cjwatson: I think this is risky. `snapName` being defined infers snapcraft.yaml is valid, but how do we tell snapcraft.yaml is invalid or missing? I have an [open question about this in Trello](https://trello.com/c/tCfftHzV/79-snapcraft-yaml-integration-1d). I'm reluctant to start this feature until this has been clarified.